Sources/Graph/set.cc

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018-2019 The Simons Foundation, Inc. - All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "set.hpp"
+
+#include <algorithm>
+#include <numeric>
+#include <sstream>
+
+namespace netket {
+
+Set::Set(int size) : n_sites_(size) {
```

Comment:
The name `Set` could be somewhat misleading. In graph theory, this structure is is sometimes called the _edgeless_ graph. `netket.graph.Edgeless` might sound a bit strange, but is it pretty unambiguous.